src/Managed.sol

```diff
@@ -122,54 +71,13 @@ abstract contract Managed is UUPSUpgradeable {
     /// @notice Sets the current admin of the contract and clears the proposed admin.
     /// @param newAdmin The admin address being set. Can be the zero address.
     function _updateAdmin(address newAdmin) internal {
-        emit AdminChanged(admin(), newAdmin);
-        _managedStorage().proposedAdmin = address(0);
+        _proposedAdmin().value = address(0);
         StorageSlot.getAddressSlot(_ADMIN_SLOT).value = newAdmin;
```

Comment:
`_updateAdmin` is used in `renounceAdmin` too which sets the admin address to address `0` and `_changeAdmin` requires that the address is not `0`. `_changeAdmin` could be used in `acceptAdmin` only, but then there would be 2 paths doing the same things and scattered across multiple files, one of them with a `require` that can't be triggered (`acceptAdmin` sets the admin to `msg.sender`) and one without the `require` because it would always be triggered (`renounceAdmin` sets the admin to `0`).